src/Views/Shared/ds-Header.cshtml

```diff
@@ -10,7 +11,7 @@
             <div class="govuk-header__content">
             @if (displayHeading)
             {
-                <a href="#" class="govuk-header__link govuk-header__link--service-name">
+                <a href="@baseUrl" class="govuk-header__link govuk-header__link--service-name">
```

Comment:
is ViewData required? 

could we use @Model.StartFormUrl, rather then placing it in ViewData